### PR TITLE
some typos fixed in step_two.md

### PR DESCRIPTION
**Pull request by lixiaobing1**
some typos fixed in step_two.md

### CODE REVIEW

File: docs/getstarted/step_two.md [hunk 1]
```diff
@@ -29,17 +29,17 @@ When you ran the command, Docker Engine:
 
 * checked to see if you had the `hello-world` software image
 * downloaded the image from the Docker Hub (more about the hub later)
-* loaded the image into the container and "ran" it
+* loaded the image into the container and "run" it
```
Comment (AkihiroSuda) [inline docs/getstarted/step_two.md]: This is not a typo

Comment (lixiaobing10051267) [inline docs/getstarted/step_two.md]: ok, has been restored.

File: docs/getstarted/step_two.md [hunk 2]
```diff
@@ -35,12 +35,12 @@ Depending on how it was built, an image might run a simple, single command and t
 
 A Docker image, though, is capable of much more. An image can start software as complex as a database, wait for you (or someone else) to add data, store the data for later use, and then wait for the next person.
 
-Who built the `hello-world` software image though? In this case, Docker did but anyone can. Docker Engine lets people (or companies) create and share software through Docker images. Using Docker Engine, you don't have to worry about whether your computer can run the software in a Docker image &mdash; a Docker container *can always run it*.
```
Comment (thaJeztah) [inline docs/getstarted/step_two.md]: don't think the `&mdash;` was a typo, as it's valid in Markdown to use

**Comment by thaJeztah** [comment]
This document have moved to the github.com/docker/docker.github.io repository, so I'll have to close this PR, but thanks again for contributing!

**Pull request closed without merging**